sdk/keyvault/keyvault-secrets/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 4.0.3 (2020-05-11)
+
+- Fixed a bug on related to the challenge based authentication that caused consecutive new authentications on new requests.
```

Comment:
Changelog updates regarding bug fixes should have a pointer to the actual bug that was reported.

Reply:
Also, this line doesnt explain very well on what the issue was. New authentications on new requests sounds just fine on the surface...